extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -319,6 +341,54 @@ export class AmpSelector extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Handles toggle action.
+   * @param {number} index
+   * @param {boolean} value
+   */
+  toggle_(index, value) {
+    // Change the selection to the next element in the specified direction.
+    // The selection should loop around if the user attempts to go one
+    // past the beginning or end.
+    const indexCurrentStatus = this.options_[index].hasAttribute('selected');
+    const indexFinalStatus =
```

Comment:
The `!!` shouldn't be necessary here, since value and indexCurrentStatus are already boolean.